lib/cli.bash

```diff
@@ -0,0 +1,108 @@
+  elif (( CURRENT == 3 )); then
+    case "$words[2]" in
+      changelog) local -a refs
+        refs=("${(@f)$(cd "$OMB"; command git for-each-ref --format="%(refname:short):%(subject)" refs/heads refs/tags)}")
+        _describe 'command' refs ;;
+      plugin) subcmds=(
+        'disable:Disable plugin(s)'
+        'enable:Enable plugin(s)'
+        'info:Get plugin information'
+        'list:List plugins'
+        'load:Load plugin(s)'
+      )
+        _describe 'command' subcmds ;;
+      pr) subcmds=('clean:Delete all Pull Request branches' 'test:Test a Pull Request')
+        _describe 'command' subcmds ;;
+      theme) subcmds=('list:List themes' 'set:Set a theme in your .zshrc file' 'use:Load a theme')
+        _describe 'command' subcmds ;;
+    esac
+  elif (( CURRENT == 4 )); then
+    case "${words[2]}::${words[3]}" in
+      plugin::(disable|enable|load))
+        local -aU valid_plugins
+
+        if [[ "${words[3]}" = disable ]]; then
+          # if command is "disable", only offer already enabled plugins
+          valid_plugins=($plugins)
+        else
+          valid_plugins=("$OMB"/plugins/*/{_*,*.plugin.zsh}(.N:h:t) "$OMB_CUSTOM"/plugins/*/{_*,*.plugin.zsh}(.N:h:t))
+          # if command is "enable", remove already enabled plugins
+          [[ "${words[3]}" = enable ]] && valid_plugins=(${valid_plugins:|plugins})
+        fi
+
+        _describe 'plugin' valid_plugins ;;
+      plugin::info)
+        local -aU plugins
+        plugins=("$OMB"/plugins/*/{_*,*.plugin.zsh}(.N:h:t) "$OMB_CUSTOM"/plugins/*/{_*,*.plugin.zsh}(.N:h:t))
+        _describe 'plugin' plugins ;;
+      theme::(set|use))
+        local -aU themes
+        themes=("$OMB"/themes/*.zsh-theme(.N:t:r) "$OMB_CUSTOM"/**/*.zsh-theme(.N:r:gs:"$OMB_CUSTOM"/themes/:::gs:"$OMB_CUSTOM"/:::))
+        _describe 'theme' themes ;;
+    esac
+  elif (( CURRENT > 4 )); then
+    case "${words[2]}::${words[3]}" in
+      plugin::(enable|disable|load))
+        local -aU valid_plugins
+
+        if [[ "${words[3]}" = disable ]]; then
+          # if command is "disable", only offer already enabled plugins
+          valid_plugins=($plugins)
+        else
+          valid_plugins=("$OMB"/plugins/*/{_*,*.plugin.zsh}(.N:h:t) "$OMB_CUSTOM"/plugins/*/{_*,*.plugin.zsh}(.N:h:t))
+          # if command is "enable", remove already enabled plugins
+          [[ "${words[3]}" = enable ]] && valid_plugins=(${valid_plugins:|plugins})
+        fi
+
+        # Remove plugins already passed as arguments
+        # NOTE: $(( CURRENT - 1 )) is the last plugin argument completely passed, i.e. that which
+        # has a space after them. This is to avoid removing plugins partially passed, which makes
+        # the completion not add a space after the completed plugin.
+        local -a args
+        args=(${words[4,$(( CURRENT - 1))]})
+        valid_plugins=(${valid_plugins:|args})
+
+        _describe 'plugin' valid_plugins ;;
+    esac
+  fi
+
+  return 0
+}
+
+compdef _omb omb
```

Comment:
`compdef` is also a Zsh feature.
```suggestion
complete -F _omb omb
```

- `_describe` should be properly defined for Bash `complete -F`. Such as `_describe() { COMPRELY+=("$(compgen -W '"$1"' -- "${COMP_WORDS[COMP_CWORD]}")"); }`.
- The array variable `words` in the body of `_omb` should be replaced by `COMP_WORDS`. Also, be careful that the index is shifted by 1 in Zsh compared to Bash; The array index of Zsh starts from 1 while that of Bash starts from 0.
- The variable `CURRENT` should be replaced by `COMP_CWORD+1`.